protos/Makefile

```diff
@@ -2,4 +2,9 @@
 go:
 	protoc --go_out=../go --go_opt=module=github.com/in-toto/attestation/go $(shell find ./in_toto_attestation -name "*.proto")
 
+# Creates the python libs in the python/ directory.
+python: 
+	mkdir -p ../python
```

Comment:
For consistency, let's take this out? If the dir is made the first time, that likely suffices.